CMakeLists.txt

```diff
@@ -106,9 +106,10 @@ else()
 	message("Normal Build")
 endif()
 
-if(LINUX)
-	message("Using direct to display extension...")
+if(LINUX AND VULKAN)
+	message("Using XCB extension...")
```

Comment:
Err?

-[Unknown]

Reply:
This stuff went through a few iterations very quickly, oops :)   (had strange issues so I tried switching to the xcb stuff but then found out what the problem with the x11 stuff was and fixed that and went back...).

I'll clean things up soon.